components/kyma-metrics-collector/pkg/process/process.go

```diff
@@ -353,25 +353,59 @@ func isSuccess(status int) bool {
 	return false
 }
 
-func isClusterTrackable(runtime *kebruntime.RuntimeDTO) bool {
-	if runtime.Status.Provisioning != nil &&
-		runtime.Status.Provisioning.State == string(kebruntime.StateSucceeded) &&
-		runtime.Status.Deprovisioning == nil {
+// Helper function which returns true if the runtime state is trackable
+func isTrackableState(state kebruntime.State) bool {
+	switch state {
+	case kebruntime.StateSucceeded, kebruntime.StateError, kebruntime.StateUpgrading, kebruntime.StateUpdating:
```

Comment:
Are these all the available states?

Reply:
There are more possible states, but only the ones provided here make the runtime trackable.